core/services/assets/AssetManager.php

```diff
@@ -169,33 +163,19 @@ public function getStylesheetAssets()
 
 
     /**
-     * @return void
-     * @since $VID:$
-     */
-    public function enqueueJsAsset($handle)
-    {
-        if($this->assets->has($handle)){
-            /** @var JavascriptAsset $script */
-            $script = $this->assets->get($handle);
-            if ($script->isRegistered()){
-                wp_enqueue_script($handle);
-            }
-        }
-    }
-
-
-    /**
-     * @return void
+     * @param string $handle
+     * @return bool
      * @since $VID:$
      */
-    public function enqueueCssSAsset($handle)
+    public function enqueueAsset($handle)
     {
         if ($this->assets->has($handle)) {
-            /** @var StylesheetAsset $script */
-            $stylesheet = $this->assets->get($handle);
-            if ($stylesheet->isRegistered()) {
-                wp_enqueue_style($handle);
+            $script = $this->assets->get($handle);
```

Comment:
minor thing, but since this could be a script or a stylesheet I wonder if it'd be better to just name this variable `$asset`?

Reply:
right... copy pasta... will fix